docs/custom.css

```
@@ -1949,3 +1953,88 @@ a.learn-sidebar-item.is-active {
   color: var(--mastra-green-accent);
 }
+
+/* ==================== ALGOLIA SEARCH PLUGIN ==================== */
+/*
+ * Helper classes consumed by @mastra/docusaurus-plugin-algolia. The plugin's
+ * UI references the `.mastra-card`, `.mastra-search-button`, `.dialog-panel`
+ * classes, which are normally provided by @mastra/docusaurus-theme. This site
+ * does not use that theme, so they are mapped here onto the docs' own design
+ * tokens. The `--algolia-*` / `--mastra-*` variables the plugin reads are
+ * defined alongside the rest of the site's design tokens above.
+ */
```

Stylelint (17.14.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)